docs/en/integrations/data-ingestion/clickpipes/index.md

```diff
@@ -180,42 +181,54 @@ If your instance region is not listed here, it will fall to the default region:
+  ClickPipes for Kafka provides `at-least-once` delivery semantics (as one of the most commonly used approaches). We'd love to hear your feedback on delivery semantics (contact form)[https://clickhouse.com/company/contact?loc=clickpipes]. If you need exactly-once semantics, we recommend using our official [`clickhouse-kafka-connect`](https://clickhouse.com/blog/real-time-event-streaming-with-kafka-connect-confluent-cloud-clickhouse) sink.
 
-- **Is there a way to handle errors or failures when using ClickPipes for Kafka ?**
+- **Is there a way to handle errors or failures when using ClickPipes for Kafka?**
 
   Yes, ClickPipes for Kafka will automatically retry case of failures when consuming data from Kafka. ClickPipes also supports enabling a dedicated error table that will hold errors and malformed data for 7 days.
 
-- **Does using ClickPipes incur an additional cost ?**
+- **Does using ClickPipes incur an additional cost?**
 
   ClickPipes is not billed separately at the moment. Running ClickPipes might generate an indirect compute and storage cost on the destination ClickHouse Cloud service like any ingest workload.
 
 - **What authentication mechanisms are supported for ClickPipes for Kafka?**
 
-  For Apache Kafka protocol data sourced, ClickPipes supports [SASL/PLAIN](https://docs.confluent.io/platform/current/kafka/authentication_sasl/authentication_sasl_plain.html) authentication with TLS encryption. For Amazon MSK ClickPipes, Redpanda, and Upstash supports [SCRAM-SHA-512](https://docs.aws.amazon.com/msk/latest/developerguide/msk-password.html) authentication.
+  For Apache Kafka protocol data sourced, ClickPipes supports [SASL/PLAIN](https://docs.confluent.io/platform/current/kafka/authentication_sasl/authentication_sasl_plain.html) authentication with TLS encryption. For Amazon MSK ClickPipes, Redpanda, and Upstash supports [SASL/SCRAM-SHA-512](https://docs.aws.amazon.com/msk/latest/developerguide/msk-password.html) authentication.
 
 - **Do you support schema registries for managed Kafka Providers?**
 
 You can also select a Schema Registry server and credentials to handle your decoding and validation (Currently only available for Confluent Cloud).
 
 - **Does ClickPipes AWS MSK support IAM authentication?**
 
-AWS MSK authentication currently only supports SASL/SCRAM-SHA-512 authentication. IAM authentication is coming soon.
+AWS MSK authentication currently only supports [SASL/SCRAM-SHA-512](https://docs.aws.amazon.com/msk/latest/developerguide/msk-password.html) authentication. IAM authentication is coming soon.
+
+- **What authentication does ClickPipes support for S3 Buckets?**
+
+You can access public buckets with no configuration, and with protected buckets you can use [IAM credentials](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html) or an [IAM Role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html).
```

Comment:
I think we should point them to this doc. It outlines how to set up the necessary permissions for S3. Otherwise there's no indication what's required.

https://clickhouse.com/docs/en/cloud/manage/security/secure-s3